src/Compilers/Core/Portable/NativePdbWriter/PdbWriter.cs

```diff
@@ -73,78 +73,81 @@ public void SerializeDebugInfo(IMethodBody methodBody, StandaloneSignatureHandle
             // Special case a hidden entry point (#line hidden applied) that would otherwise have no debug info.
             // This is to accommodate for a requirement of Windows PDB writer that the entry point method must have some debug information.
             bool isKickoffMethod = methodBody.StateMachineTypeName != null;
-            bool emitDebugInfo = isKickoffMethod || !methodBody.SequencePoints.IsEmpty ||
+            bool emitAllDebugInfo = isKickoffMethod || !methodBody.SequencePoints.IsEmpty ||
                 methodBody.MethodDefinition == (Context.Module.DebugEntryPoint ?? Context.Module.PEEntryPoint);
 
-            if (!emitDebugInfo)
+            var compilationOptions = Context.Module.CommonCompilation.Options;
+
+            // We need to avoid emitting CDI DynamicLocals = 5 and EditAndContinueLocalSlotMap = 6 for files processed by WinMDExp until 
+            // bug #1067635 is fixed and available in SDK.
```

Comment:
This bug is never getting fixed, right? 😁

Reply:
Probably not.